hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -2218,3 +2218,77 @@ def hpat_pandas_series_median_impl(self, axis=None, skipna=True, level=None, num
         return numpy.median(self._data)
 
     return hpat_pandas_series_median_impl
+
+
+@overload_method(SeriesType, 'dropna')
+def hpat_pandas_series_dropna(self, axis=0, inplace=False):
+    """
+    Pandas Series method :meth:`pandas.Series.dropna` implementation.
+
+    .. only:: developer
+
+       Tests: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_dropna_axis1
```

Comment:
I propose to use pattern reflected all tests:
`python -m hpat.runtests -k hpat.tests.test_series.TestSeries.test_series_dropna*`
`-k` lets to run it in command line.

Reply:
Done